### PR TITLE
Rewrite manipulation tests not to use `numpy_cupy_raises`

### PR DESCRIPTION
**Pull request by asi1024**
Ready for #3098.

### CODE REVIEW

File: cupy/core/internal.pyx [hunk 1]
```diff
@@ -278,7 +278,9 @@ cpdef float from_float16(uint16_t v):
 cdef inline int _normalize_order(order, cpp_bool allow_k=True) except? 0:
     cdef int order_char
     order_char = b'C' if len(order) == 0 else ord(order[0])
-    if allow_k and (order_char == b'K' or order_char == b'k'):
+    if order_char == b'K' or order_char == b'k':
+        if not allow_k:
+            raise ValueError('order \'K\' is not permitted')
```
Comment (niboshi) [inline cupy/core/internal.pyx]: How about separating this fix?

Comment (asi1024) [reply cupy/core/internal.pyx]: Splitted into #3123.

**Comment by niboshi** [comment]
Jenkins, test this please

File: tests/cupy_tests/manipulation_tests/test_dims.py [hunk 2]
```diff
@@ -134,24 +139,19 @@ def test_expand_dims_tuple_axis(self, xp):
...
-                (1, 5),
-        ]:
-            with self.assertRaises(numpy.AxisError):
-                xp.expand_dims(a, axis)
-        return xp.array(True)
+        for xp in (numpy, cupy):
+            a = testing.shaped_arange((2, 2, 2), xp)
+            for axis in [(1, -6), (1, 5)]:
+                with pytest.raises(numpy.AxisError):
+                    xp.expand_dims(a, axis)
```
Comment (niboshi) [inline tests/cupy_tests/manipulation_tests/test_dims.py]: Remove `xp` argument.

File: tests/cupy_tests/manipulation_tests/test_dims.py [hunk 3]
```diff
@@ -134,24 +139,19 @@ def test_expand_dims_tuple_axis(self, xp):
...
-    @testing.numpy_cupy_array_equal()
     def test_expand_dims_repeated_axis(self, xp):
-        a = testing.shaped_arange((2, 2, 2), xp)
-        with self.assertRaises(ValueError):
-            xp.expand_dims(a, (1, 1))
-        return xp.array(True)
+        for xp in (numpy, cupy):
+            a = testing.shaped_arange((2, 2, 2), xp)
+            with pytest.raises(ValueError):
+                xp.expand_dims(a, (1, 1))
```
Comment (niboshi) [inline tests/cupy_tests/manipulation_tests/test_dims.py]: Ditto.

**Comment by asi1024** [comment]
@niboshi Fixed. PTAL.

**Comment by niboshi** [comment]
Jenkins, test this please

**Pull request merged**